dataactbroker/handlers/fileHandler.py

```diff
@@ -702,7 +699,7 @@ def handleEmptyResponse(self, job, valJob):
             valJob.number_of_errors = 0
             valJob.number_of_warnings = 0
             valJob.filename = None
-            jobDb.session.commit()
+            sess.commit()
```

Comment:
Good call. That's, uh, a lot of commits, especially considering that the `mark_job_status` function being called is also committing. Will remove some of these. 

I do think that a longer-term fun project would be to commit based on a sensible unit of work. The committing-at-the-bottom-of-every-function thing is an artifact of the old db setup.